guides/source/3_2_release_notes.md

```diff
@@ -327,7 +327,7 @@ Active Record
 
 * Implemented `ActiveRecord::Relation#explain`.
 
-* Implements `AR::Base.silence_auto_explain` which allows the user to selectively disable automatic EXPLAINs within a block.
+* Implements `ActiveRecord::Base.silence_auto_explain` which allows the user to selectively disable automatic EXPLAINs within a block.
```

Comment:
@senny thanks for review. I have committed changes that i think should really be helpful to documentation of framework. And thanks for the [ci skip] tip will this in mind.